froide/settings.py

```diff
@@ -501,7 +501,7 @@ def is_pkce_required(client_id):
             "read:email": _("Read user email"),
             "read:request": _("Read your (private) requests"),
             "make:request": _("Make requests on your behalf"),
-            "upload:message": _("Upload postal messages"),
+            "create:message": _("Add messages and attachments to user's request"),
```

Comment:
Could this lead to weird behaviours for existing keys / apps that don't have this scope?

Reply:
Yes. I suppose it's only used by the scanner app though.

Reply:
Only staff users have tokens with this scope.